cmake/hipify.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
```

Comment:
QQ: Why do we need hipify?

IIUC, the current version of flash-attn does not support ROCm.

Reply:
This was here initially to make it easier to keep the CMake build system (especially `utils.cmake`) as similar to the vLLM repo as possible. But I'll delete this file